micro-ui/web/micro-ui-internals/packages/modules/sandbox/src/configs/UICustomizations.js

```
@@ -91,182 +91,182 @@
+              } else {
+                window.history.pushState(
+                  null,
+                  "",
+                  `/${window.contextPath}/employee/workbench/mdms-search-v2?moduleName=${value?.split(".")?.[0]}&masterName=${
+                    value?.split(".")?.[1]
+                  }&from=sandbox&module=${column?.module}`
+                );
+                const navEvent = new PopStateEvent("popstate");
+                window.dispatchEvent(navEvent);
+              }
+            };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)